src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -100,6 +100,7 @@
     public static final String DEVICE_PARAM_AWS_REGION = "awsRegion";
     public static final String DEVICE_MQTT_NAMESPACE = "mqtt";
     public static final String DEVICE_SPOOLER_NAMESPACE = "spooler";
+    public static final String FLEET_STATUS_CONFIG_TOPICS = "status";
```

Comment:
status seems too generic IMO. Maybe just "fleetStatus" or "coreStatus"? Other suggestions would be great.

Reply:
I think "fleetStatus" seems reasonable.